text/entities/0256-entities-data-model.md

```diff
@@ -0,0 +1,790 @@
+
+## Entity Identification
+
+_This section is a supplementary guideline and is not part of logical data model._
+
+The data model defines the structure of the entity Id field. This section explains
+how the Id field is computed.
+
+### LID, GID and IDCONTEXT
+
+All entities have a local ID (LID) and a global ID (GID).
+
+The LID is unique in a particular identification context, but is not necessarily globally
+unique. For example a process entity's LID is its PID number and process start time.
+The (PID,StartTime) pair is unique only in the context of a host where the process runs
+(and the host in this case is the identification context).
+
+The GID of an entity is globally unique, in the sense that for the entire set of entities
+in a particular telemetry store no 2 entities exist that have the same GID value.
+
+The GID of an entity E is defined as:
+
+`GID(E) = { LID(E), GID(IDCONTEXT(E)) }`
+
+Where `IDCONTEXT(E)` is the identification context in which the LID of entity E is unique.
+The value of `IDCONTEXT(E)` is an entity itself, and thus we can compute the GID value of it too.
+
+In other words, the GID of an entity is a union of its LID and the GID of its
+identification context. Note: GID(E) is a map of key/value attributes.
+
+The enrichment process often is responsible for determining the value of `IDCONTEXT(E)`
+and for computing the GID according to the formula defined above, although the GID may
+also be produced at once by the telemetry source (e.g. by OTel SDK) without requiring
+any additional enrichment.
+
+### Semantic Conventions
+
+OpenTelemetry semantic conventions will be enhanced to include entity definitions for
+well-known entities such as Service, Process, Host, etc.
+
+For well-known entity types LID(E) is defined in OTel semantic conventions per
+entity type. The value of LID is a map of key/value attributes. For example,
+for entity of type "process" the semantic conventions define LID as 2 attributes:
+
+```json5
+{
+  "process.pid": $pid,
+  "process.start_time": $starttime
+}
+```
+
+For custom entity types (not defined in OTel semantic conventions) the end-user is
+responsible for defining their custom semantic conventions in a similar way.
+
+The entity information producer is responsible for determining the identification
+context of each entity it is producing information about.
+
+In certain cases, where only one possible IDCONTEXT definition is meaningful, the
+IDCONTEXT MAY be defined in the semantic conventions. For example Kubernetes nodes
+always exist in the identifying context of a Kubernetes cluster. The semantic convention
+for "k8s.node" and "k8s.cluster" can prescribe that the IDCONTEXT of entity of type
+"k8s.node" is always an entity of type "k8s.cluster".
+
+Important: semantic conventions are not expected to (and normally won't) prescribe
+the complete GID composition.
+Semantic conventions SHOULD prescribe LID and MAY prescribe IDCONTEXT, but GID
```

Comment:
These are normative statements in a supplementary guideline. Restructuring to move these into the non-supplementary guideline sections will help it from being missed and prevent confusion.

Reply:
You are right. This should be in a normative section.

Reply:
Fixed.